bigflow/scaffold/scaffold_templates.py

```diff
@@ -159,6 +159,62 @@ def run(self, runtime):
                                'dags_bucket': '{dags_bucket}'}})
 '''
 
+bq_workflow_template = """
```

Comment:
Why you've changed files related with scaffold?

Reply:
1. It's much easier to relate to the single file in doc
2. It's more elegant, easier to understand what is going on in the example
3. It presents that code setup is up to you - you can start from a simple setup with a single file
4. It's easier to maintain for us

Reply:
Ok 👍